common/ideologies/imperia_leader_ideologies.txt

```diff
@@ -0,0 +1,38 @@
+﻿imperia_ideology_modernizer = { # Clone of VTM Modernizer for use with our historical characters.
+	icon = "gfx/interface/icons/ideology_icons/ideology_leader/unused/ideology_leader_particularist.dds"
+
+	character_ideology = yes
+
+	lawgroup_bureaucracy = {
+		law_elected_bureaucrats = disapprove
+		law_hereditary_bureaucrats = neutral
+		law_appointed_bureaucrats = approve
+		law_meritocratic_bureaucracy = approve
+	}
+
+	lawgroup_army_model = {
+		law_mass_conscription = approve
+		law_professional_army = approve
+		law_national_militia = neutral
+		law_peasant_levies = disapprove
+	}
+
+	lawgroup_economic_system = {
+		law_command_economy = strongly_disapprove
+		law_cooperative_ownership = strongly_disapprove
+		law_interventionism = approve
+		law_agrarianism = neutral
+		law_traditionalism = disapprove	
+		law_laissez_faire = disapprove
+		law_industry_banned = strongly_disapprove
+	}
+
+	possible = {
+		# Intentionally left empty
+	}
```

Comment:
Isn't leaving it intentionally empty potentially problematic? Why not just copy the possible from our merge file with vtm?

Reply:
@Danarca can you weigh in on this

Reply:
I... _Think_ leaving it empty will allow it to pop up everywhere. If you want to use it only in scripted situations, I'd try giving it an `always = no`